sdk/core/azure-core/src/main/java/com/azure/core/util/logging/ClientLogger.java

```diff
@@ -37,6 +38,7 @@
  */
 public class ClientLogger {
     private final Logger logger;
+    private LogLevel rootLogLevel;
```

Comment:
This introduces potential multi-threading concerns. It'd be better to leave this as a local variable when it is used.

Reply:
Done with loading sys env everytime.